_tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api.md

```diff
@@ -49,90 +52,289 @@ GET _remotestore/stats/<index_name>
+| `last_successful_upload_timestamp`| The timestamp, in milliseconds, since the last successful translog file uploaded to remore-backed storage. |
+| `total_uploads.started` | The total number of attempted translog upload syncs to remote-backed storage. |
+| `total_uploads.failed` | The total number of failed translog upload syncs to remote-backed storage.   |
+| `total_uploads.succeeded` | The total number of successful translog upload syncs to remote-backed storage.  |
+| `total_upload_size.started_bytes` | The total number of bytes of translog files actively being downloaded from remote-backed storage. |
+| `total_upload_size.succeeded_bytes` | The total number of bytes of translog files successfully uploaded to remote-backed storage. |
+|`total_upload_size.failed_bytes` | The total number of bytes of translog files that failed to uploaded to remote-backed storage. |
+| `total_upload_time_in_millis` | The total time spent, in milliseconds, uploading translog files to remote-backed storage. |
+| `upload_size_in_bytes.moving_avg` | The average size of translog data, in bytes, uploaded in the last *N* downloads. *N* is defined in the `remote_store.moving_average_window_size` setting. |
+| `upload_speed_in_bytes_per_sec.moving_avg` | The average speed of translog uploads, in bytes per second, for the last *N* uploads. *N* is defined in the `remote_store.moving_average_window_size` setting.    |
+| `upload_time_in_millis.moving_avg` | The average time taken by a single translog upload, in milliseconds, since the last *N* uploads. *N* is defined in `remote_store.moving_average_window_size` setting.    |
+
+The `translog.download` object contains the following fields:
+
+|Field	|Description	|
+|:---	|:---	|
+| `last_successful_download_timestamp` | The timestamp, in milliseconds, since the last successful translog file uploaded to remote-backed storage. |
+| `total_downloads.succeeded` | The total number of successful translog download syncs from remote-backed storage. |
+| `total_download_size.succeeded_bytes` | The total number of bytes of translog files successfully uploaded from remote-backed storage.  |
+| `total_download_time_in_millis` | The total time spent, in milliseconds, downloading translog files from remote-backed storage.  |
+| `download_size_in_bytes.moving_avg`  | The average size of translog data, in bytes, downloaded in the last *N* downloads. *N* is defined in the `remote_store.moving_average_window_size` setting.    |
+| `download_speed_in_bytes_per_sec.moving_avg` | The average speed of translog downloads, in bytes per second, for the last *N* uploads. *N* is defined in the `remote_store.moving_average_window_size` setting.   |
+| `download_time_in_millis.moving_avg` |  The average time taken by a single translog download, in milliseconds, since the last *N* uploads. *N* is defined in `remote_store.moving_average_window_size` setting.  |
```

Comment:
```suggestion
| `download_time_in_millis.moving_avg` |  The average amount of time taken by a single translog download, in milliseconds, since the last *N* uploads. *N* is defined in the `remote_store.moving_average_window_size` setting.  |
```